introduction/motivation.md

```diff
@@ -0,0 +1,40 @@
+# Motivation to course
+
+Maybe you already tried to jump into Javascript and 
+got stuck in the middle of all these super shiny libraries 
+you absolutely need to know before you can write anything meaningful. 
+
+Surprisingly, there is even article about it: [How it feels to learn JavaScript in 2016](https://hackernoon.com/how-it-feels-to-learn-javascript-in-2016-d3a717dd577f)
+
+So you might be asking yourself - why learn JavaScript?
+
+## Why JavaScript
+
+**It's everywhere!** not only in your browser:
+  - [Electron](http://electron.atom.io/) - framework for desktop apps, 
+    used already by several companies, e.g. guess what's behind Slack... 
+  - [React Native](https://facebook.github.io/react-native/) for native mobile applications
```

Comment:
We can add more examples of using JS, e.g. Facebook app for React Native or Netflix as company that has Node.js backend.